app/views/layouts/application.html.erb

```diff
@@ -13,9 +13,21 @@
   <body>
     <%= render 'layouts/header' %>
 
-    <!-- Messages -->
+    <% # Messages %>
```

Comment:
Szerintem nem érdemes kommentet hagynunk a kódban. Kérlek ezt vegyük ki!

E helyett csinálhatnánk egy partial-t, ami egyértelművé teszi, hogy mit is csinál az itt lévő html kód.
Ehhez arra lenne szükség, hogy a kódot mozgassuk át egy `_flash_messages.html.erb` vagy hasonló nevű fájlba.
Majd itt csak rendereljük azt ki:
```
<%= render 'flash_messages' %>
```